mongo/src/test/java/org/axonframework/extensions/mongo/eventsourcing/eventstore/MongoOptionsFactoryTest.java

```diff
@@ -37,15 +39,17 @@ void setUp() {
 
     @Test
     void testCreateMongoOptions_defaults() {
-        MongoClientOptions options = factory.createMongoOptions();
-        MongoClientOptions defaults = MongoClientOptions.builder().build();
+        MongoClientSettings options = factory.createMongoOptions();
+        MongoClientSettings defaults = MongoClientSettings.builder().build();
 
-        assertEquals(defaults.getMaxWaitTime(), options.getMaxWaitTime());
-        assertEquals(defaults.getSocketTimeout(), options.getSocketTimeout());
-        assertEquals(defaults.getConnectionsPerHost(), options.getConnectionsPerHost());
-        assertEquals(defaults.getConnectTimeout(), options.getConnectTimeout());
-        assertEquals(defaults.getThreadsAllowedToBlockForConnectionMultiplier(),
-                     options.getThreadsAllowedToBlockForConnectionMultiplier());
+        assertEquals(defaults.getConnectionPoolSettings().getMaxWaitTime(TimeUnit.MILLISECONDS), options.getConnectionPoolSettings().getMaxWaitTime(TimeUnit.MILLISECONDS));
+        assertEquals(defaults.getSocketSettings().getConnectTimeout(TimeUnit.MILLISECONDS), options.getSocketSettings().getConnectTimeout(TimeUnit.MILLISECONDS));
+        assertEquals(defaults.getConnectionPoolSettings().getMaxSize(), options.getConnectionPoolSettings().getMaxSize());
+        // TODO connect time and socket timeout is identical
```

Comment:
Are you certain these are identical? If so, do you per chance have any material on this to reference for me?

Reply:
There was no document that described the changes made and how to use the new methods in the MongoDb docs so I had to go comparing the old docs with the new docs.

http://mongodb.github.io/mongo-java-driver/4.0/apidocs/mongodb-driver-core/com/mongodb/connection/ConnectionPoolSettings.html
http://mongodb.github.io/mongo-java-driver/4.0/apidocs/mongodb-driver-core/com/mongodb/connection/SocketSettings.html
http://mongodb.github.io/mongo-java-driver/3.12/javadoc/com/mongodb/MongoClientOptions.html